docs/getting-started/continuous_integration.md

```diff
@@ -221,16 +221,22 @@ The results of CIFuzz can be found in two different places.
 ![Finding fuzzer output](../images/run_fuzzers.png)
 
 
+
 *  Artifacts:
-    1. When a crash is found by CIFuzz the Upload Artifact event is triggered.
-    1. This will cause a pop up in the right hand corner, allowing
-    you to download a zip file called `artifacts`.
-    1. `artifacts` contains two files for each crash:
-        * A test case that can be used to reproduce the crash.
-        * The sanitizer stack trace of the crash.
+When the fuzzer crashes the input file that causes the crash is uploaded as an
+artifact.
+To download the artifact, do the following steps:
+1. Click on the summary from the run, as illustrated in the screenshot below:
+
+![github-actions-summary]
+
+1. Click on the artifact you wish to download from the summary page, as
```

Comment:
I thought markdown would number this itself, but i'll change it (according to the rich view I'm wrong, but I may actually be right since we use jenkins)

Reply:
Ah, interesting!